lgc/interface/lgc/Pipeline.h

```diff
@@ -115,6 +115,7 @@ struct Options {
   unsigned nggPrimsPerSubgroup;        // How to determine NGG prims per subgroup
   unsigned shadowDescriptorTable;      // High dword of shadow descriptor table address, or
                                        //   ShadowDescriptorTableDisable to disable shadow descriptor tables
+  unsigned robustAccess;               // If set, support VK_EXT_robustness2
```

Comment:
(Sorry if this is a repeat; I think the github UI lost my previous attempt.)
This is the LGC interface, so I would prefer to describe things in terms of what they do, instead of referring to Vulkan or SPIR-V. So, could we call this field "allowNullBufferDesc", and comment it "Allow and give defined behavior on a null buffer descriptor"?

Reply:
VK_EXT_robustness2 requires to handle out of bound store and load and null descriptor. So I think the option may not be null-descriptor-specified, right?

Reply:
Ah, ok, then the name "robustAccess" makes more sense. Perhaps the comment could explain, e.g. "Give defined behavior for null buffer descriptor and out-of-range buffer load and store".

Reply:
Fixed the comment.